data/nav.yml

```diff
@@ -429,7 +429,30 @@
           path: "apis/rest-api"
         - name: "Limits"
           path: "apis/rest-api/limits"
-        - name: "Pipelines REST API"
+        - name: "Agent"
+          children:
+            - name: "Overview"
+              path: "apis/agent-api"
+            - name: "Metrics"
+              path: "apis/agent-api/metrics"
+        - name: "Analytics"
+          children:
+            - name: "Flaky tests"
+              path: "apis/rest-api/analytics/flaky-tests"
+            - name: "Runs"
+              path: "apis/rest-api/analytics/runs"
+            - name: "Suites"
+              path: "apis/rest-api/analytics/suites"
+            - name: "Tests"
+              path: "apis/rest-api/analytics/tests"
+        - name: "Organizations"
+          children:
+            - name: "Organizations"
```

Comment:
Also as part of addressing the problem described in my first comment, consider changing this to "Overview" to avoid the "Organizations > Organizations" issue in the page nav.